examples/EventV1/EventV2.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I reworked Portal to push events that look like this into the RabbitMQ queue. I am inlining the config and data into the `source` field as JSON and CSV strings respectively. This is everything required for NAE to perform an analysis.